src/Assets/Styles/components/_toggle.scss

```diff
@@ -0,0 +1,45 @@
+.js-toggle {
+  @include govuk-font($size: 19, $weight: 'bold');
+  @include govuk-responsive-margin(6, "bottom");
+  background-color: govuk-colour("grey-3");
+  background-image: url('/images/option-select-toggle-sprite.png');
+  background-position: right 5px;
+  background-repeat: no-repeat;
+  border: none;
+  color: $govuk-text-colour;
+  display: none;
+  padding: 8px;
+  text-align: left;
+  text-decoration: none;
+  width: 100%;
+
+  &.is-toggled {
+    background-position: right -75px;
+  }
+
+  .js-enabled & {
+    display: block;
+
+    // hide for desktop only
+    @include mq ($from: desktop) {
+      display: none;
+    }
+  }
+}
+
+.js-toggle-content {
+  display: block;
+
+  .js-enabled & {
+    display: none;
+
+    // always show for desktop
+    @include mq ($from: desktop) {
```

Comment:
Does this mean that any browser that doesn't support media queries won't see filters? Probably only IE8

Reply:
Correct. Do we support IE8? https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices